src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -31,6 +31,7 @@ type Props = {
     onTaskAssignmentUpdate?: Function,
     onTaskCreate?: Function,
     onTaskDelete?: Function,
+    onTaskModalClose?: Function,
```

Comment:
Fine for now, but for new entries should start adding explicit flow types so () => void kind

Reply:
() => any is equivalent to Function, right?